one-pager-maker/src/App.tsx

```diff
@@ -36,18 +36,29 @@ function App() {
             <BrowserRouter>
                 {user ? (
                     <>
-                        <nav>
-                            <ul id="nav">
-                                <li>
-                                    <Link to="/">一覧</Link>
-                                </li>
-                                <li>
-                                    <Link to="/edit">編集</Link>
-                                </li>
-                                <li>
-                                    <Button onClick={() => { signOut(auth) }}>ログアウト</Button>
+                        <nav className="relative px-4 py-4 flex justify-between items-center bg-white">
+                            <Link className="text-3xl font-bold leading-none" to="/">
+                                <FiFileText className='inline' />
+                                <a className="p-1">One Pager Maker</a>
+                            </Link>
+                            <div className="lg:hidden">
+                                <button className="navbar-burger flex items-center text-blue-600 p-3">
+                                    <svg className="block h-4 w-4 fill-current" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg">
+                                        <title>Mobile menu</title>
+                                        <path d="M0 3h20v2H0V3zm0 6h20v2H0V9zm0 6h20v2H0v-2z"></path>
+                                    </svg>
+                                </button>
+                            </div>
+                            <ul className="hidden absolute top-1/2 left-1/2 transform -translate-y-1/2 -translate-x-1/2 lg:flex lg:mx-auto lg:flex lg:items-center lg:w-auto lg:space-x-6">
+                                <li><Link className="text-sm text-gray-400 hover:text-gray-500" to="/">Home</Link></li>
+                                <li className="text-gray-300">
+                                    <svg xmlns="http://www.w3.org/2000/svg" fill="none" stroke="currentColor" className="w-4 h-4 current-fill" viewBox="0 0 24 24">
+                                        <path strokeLinecap="round" strokeLinejoin="round" strokeWidth="2" d="M12 5v0m0 7v0m0 7v0m0-13a1 1 0 110-2 1 1 0 010 2zm0 7a1 1 0 110-2 1 1 0 010 2zm0 7a1 1 0 110-2 1 1 0 010 2z" />
```

Comment:
Ellipsis icon might be misunderstood as menu button, so using other symbol like `|` would be better.

Reply:
Change |